nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -154,6 +154,18 @@
             .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
             .build();
 
+    static final PropertyDescriptor CUSTOM_ORDERBY_COLUMN = new PropertyDescriptor.Builder()
+            .name("gen-table-custom-orderby-column")
+            .displayName("Custom ORDER BY Column")
+            .description("The name of a column to be used for ordering the results if Max-Value Columns are not provided and partitioning is enabled. This property is ignored if either "
+                    + "Max-Value Columns is set or Partition Size > 0. NOTE: If neither Max-Value Columns nor Custom ORDER BY Column is set, and Partition Size > 0, then depending on the "
```

Comment:
Testing this out, everything look good, one minor comment/question - I think the statement about being ignored if max-value column is set or partition size is > 0 may not be totally right? should it be when partition size is = 0 ?

Reply:
Good catch, this is bass-ackwards :) Will update